src/test/java/org/broadinstitute/hellbender/utils/recalibration/BQSRGathererUnitTest.java

```diff
@@ -0,0 +1,136 @@
+package org.broadinstitute.hellbender.utils.recalibration;
+
+import com.google.common.collect.ImmutableList;
+import org.broadinstitute.hellbender.utils.report.GATKReport;
+import org.broadinstitute.hellbender.utils.report.GATKReportTable;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.PrintStream;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public final class BQSRGathererUnitTest extends BaseTest {
```

Comment:
These tests should maybe be moved into `RecalibrationReportUnitTest` because there isn't actually anything called BQSRGatherer anymore.